src/sentry/plugins/base/v2.py

```diff
@@ -349,15 +349,18 @@ def get_tags(self, event, **kwargs):
         """
         return []
 
-    def get_event_preprocessors(self, **kwargs):
+    def get_event_preprocessors(self, data, **kwargs):
```

Comment:
so my original intention was this could execute once to collect all registered preprocessors (and thus not be bound to a project). It's not a big deal either way since we'll probably throw away this API, just FYI.